bin/bank/pycbc_brute_bank

```diff
@@ -270,15 +271,35 @@ class GenUniformWaveform(object):
         self.md = q._data[-100:]
         self.md2 = q._data[0:100]
 
-    def generate(self, **kwds):
+    def generate(self, **kwds):  
         kwds.update(fdict)
-        if kwds['approximant'] in pycbc.waveform.fd_approximants():
-            ws = pycbc.waveform.get_fd_waveform(delta_f=self.delta_f,
+        if kwds['approximant'] in pycbc.waveform.fd_approximants():   
+            if args.cut_wavelength:
+                import numpy
```

Comment:
Shouldn't need to reimport numpy

Reply:
Move the determination of the waveform length and corresponding flow to top of this function so the result can be used by the remainder with less code duplication.